doc/source/devel/governance.rst

```diff
@@ -0,0 +1,156 @@
+responsibilities to ensure the smooth running of the project. SC members are
+expected to participate in strategic planning, approve changes to the
+governance model, and make decisions about funding granted to the project
+itself. (Funding to community members is theirs to pursue and manage.) The
+purpose of the SC is to ensure smooth progress from the big-picture
+perspective. Changes that impact the full project require analysis informed by
+long experience with both the project and the larger ecosystem. When the core
+developer community (including the SC members) fails to reach such a consensus
+in a reasonable timeframe, the SC is the entity that resolves the issue.
+
+Steering Council members appear as team members on the `Nibabel Steering
+Council Team page
+<https://github.com/orgs/nipy/teams/nibabel-steering-council/members>`_ and
+can be messaged ``@nipy/nibabel-steering-council``. Core
+
+Decision Making Process
+=======================
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project
+`mailing list <https://mail.python.org/mailman/listinfo/neuroimaging>`_
+and the `issue tracker <https://github.com/nipy/nibabel/issues>`_.
+Occasionally, sensitive discussion may occur on a private list.
+
+Decisions should be made in accordance with our :ref:`mission_and_values`.
+
+Nibabel uses a *consensus seeking* process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+Core developers are expected to distinguish between fundamental objections to a
+proposal and minor perceived flaws that they can live with, and not hold up the
+decision making process for the latter.  If no option can be found without
+an objection, the decision is escalated to the SC, which will itself use
+consensus seeking to come to a resolution. In the unlikely event that there is
+still a deadlock, the proposal will move forward if it has the support of a
+simple majority of the SC. Any proposal must be described by a Nibabel :ref:`biap`.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction
+  of a sentence (but no change of the Nibabel landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or
+  requested changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core
+  developer on the issue or pull-request page (lazy consensus).
```

Comment:
A requirement for two reviewers is a high bar for nibabel, historically. Maybe something like:

```suggestion
- **Code changes and major documentation changes** require approval by *two*
  reviewers, including at least one core developer, *and* no disagreement or
  requested changes by a core developer on the issue or pull-request page
  (lazy consensus). If a core developer calls for a second review, they may
  merge after a "reasonable" interval.
```

Reply:
I changed the two to "one" in the modified version.  Is that OK?

Reply:
Yes, that's fine.